src/Persistence/Mapping/AbstractClassMetadataFactory.php

```diff
@@ -35,30 +35,22 @@
  */
 abstract class AbstractClassMetadataFactory implements ClassMetadataFactory
 {
-    /**
-     * Salt used by specific Object Manager implementation.
-     *
-     * @var string
-     */
-    protected $cacheSalt = '__CLASSMETADATA__';
+    /** Salt used by specific Object Manager implementation. */
+    protected string $cacheSalt = '__CLASSMETADATA__';
 
-    /** @var CacheItemPoolInterface|null */
-    private $cache;
+    private ?CacheItemPoolInterface $cache = null;
```

Comment:
Shouldn't our CS fix this to…

```suggestion
    private CacheItemPoolInterface|null $cache = null;
```

…?

Reply:
Ah I wrongly assumed that it didn't because we were using an old version, but it must be because of the php version that I neglected to bump in the config file. Hang on.

Reply:
Indeed, we should maybe bump the CS package version on the lower branches first.

Reply:
Yes, the bulk of the change comes from the php version upgrade though.